scripts/CSSRE/lib/py/kafka.py

```diff
@@ -81,23 +81,24 @@ def run_kafka_topics_script(self, namespace, kafka, topic="", filter="") :
     Parameters:
       namespace (str): Namespace of the kafka.
       kafka (str): Name of the kafka cluster.
+      service (str): Name of the service being used to run oc exec against StatefulSet or StrimziPodSet.
```

Comment:
Where do I add the line of code that actually sets this? Do I need to write something like above at line 5?